### DISCUSSION TITLE
Testa non appare in nessun documento

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#155] [Richieste di aiuto] [answered]
Comunque il problema relativo alla testata scolastica che non appare più in nessun documento rimane invariato ,nonostante i mille tentativi nell'eseguire i consigli dati .

**Comment by manuelbarberi** [comment, marked as answer]
Prova a ricaricare la testata scolastica e esegui la pulizia della cache.

Dopo aver caricato la testata recati su 
/public/install/clean.txt e modificalo in clean.php . Esegui il comando cliccandoci sopra e dopo aver pulito la cache rimuovi il file clean.php  lasciando clean.txt
